pavelib/bok_choy.py

```diff
@@ -130,3 +165,25 @@ def bokchoy_coverage():
     sh("coverage html --rcfile={}".format(coveragerc))
     sh("coverage xml --rcfile={}".format(coveragerc))
     sh("coverage report --rcfile={}".format(coveragerc))
+
+
+@task
+def bokchoy_coverage():
+    """
+    Generate coverage reports for bok-choy tests
+    """
+    parse_coverage(
+        Env.BOK_CHOY_REPORT_DIR,
+        Env.BOK_CHOY_COVERAGERC
+    )
+
+
+@task
+def a11y_coverage():
+    """
+    Generate coverage reports for a11y tests
```

Comment:
Maybe add a comment here about how the coverage is a bit of a proxy, rather than actual coverage?